### ISSUE TITLE
Disallow request.addfinalizer in fixtures

### ISSUE THREAD

**Issue by m-burst** [m-burst/flake8-pytest-style#20]
# Rule request

## Description

Disallow `request.addfinalizer` in favor of `yield`-fixtures.

Bad:

```py
@pytest.fixture()
def my_fixture(request):
    resource = acquire_resource()
    request.addfinalizer(lambda: resource.release())
    return resource
```

Good:

```py
@pytest.fixture()
def my_fixture():
    resource = acquire_resource()
    yield resource
    resource.release()
```

## Rationale

* to enforce consistency
* `yield`-fixtures are more simple and readable
